src/main/resources/schema/schema

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://json-schema.org/draft-07/schema#",
-  "$id": "https://json-schema.org/draft-07/schema#",
+  "$id": "https://json-schema.org/draft-07/schema",
```

Comment:
`Validator` uses `$id` value as the key when caching meta-schemas (see ln159). The trailing # throws off the resolver.